kymatio/scattering2d/backend/torch_skcuda_backend.py

```diff
@@ -210,14 +133,14 @@ def __init__(self):
     def GET_BLOCKS(self, N):
         return (N + self.CUDA_NUM_THREADS - 1) // self.CUDA_NUM_THREADS
 
-    def __call__(self, input):
-        if not input.is_cuda:
+    def __call__(self, x):
+        if not x.is_cuda:
             raise RuntimeError('Use the torch backend for cpu tensors!')
```

Comment:
what about numpy backend? Maybe you want to edit to `'Use the torch backend (without skcuda) for cpu tensors!'`